src/jdk.compiler/share/classes/com/sun/tools/javac/resources/javac.properties

```diff
@@ -292,6 +292,10 @@ javac.opt.X=\
     Print help on extra options
 javac.opt.help=\
     Print this help message
+javac.opt.help.lint=\
+    Print the supported keys of option -Xlint
```

Comment:
Suggest: `Print the supported keys for -Xlint`
This does not affect the CSR.

Reply:
Fixed.